ereolen_support/js/clearFields.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file clearFields.js
+ *   Clear fields on form change.
+ */
+(function () {
+  Drupal.behaviors.ereolenClearFields = {
+    attach: function (context, settings) {
+      'use strict';
+
+      let problem = document.getElementById("edit-ereolen-support-problem");
+      clearFields();
```

Comment:
Minor: Maybe switch the lines so the `let` statement:

```
clearFields();

let problem = document.getElementById("edit-ereolen-support-problem");
problem.addEventListener('change', function(){
        clearFields();
});
```

Reply:
Changed